data/dungeons.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
Possible suggestion if you want to implement it :)

I see more value in improving `RotationData` here;

We could add `MasterAvailable` as a key to a `RotationEntry` along with `name`, `location` and `comments` and that would essentially represent the same data we are trying to store here.

In terms of what you'd have to add, you'd have to change the `dungeons.json` file to have the same structure as whatever is described in `internal/data/rotation_data.go`, so I would:

- Represent the `dungeons.json` data in the same structure as the other JSON files (essentially the structure that's in `rotation_data.go`
- Add `MasterAvailable` to `RotationEntry` in `rotation_data.go`
- You can then use `MasterAvailable` by calling `ReadContentRotationData` as you would for the other commands that use data

Implementing it this would also remove all the `featured_data` code

Reply:
Yeah, it didn't feel right adding the separate data structure, but you weren't around at the time for me to bounce ideas off of XD. I also didn't want to have a bunch of unused entries in `dungeons.json` (gear list, location list, gear rotation, rotation complete). I tried to see if Golang had a "optional field" (like what Typescript has), but sadly no luck.

I'll take your suggestions and make adjustments

Reply:
Oh, wait, there might be a way to have optional JSON field. Looking into it now

Reply:
FYI, I did this by adding `omitempty` to the JSON parsing. Seems to work well. Lemme know if you'd like this removed and I'll just add the empty fields to `dungeons.json`

Reply:
Yeah, `omitempty` is the way :) Sorry, should have mentioned that